### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by devopseasylearning**
Update docker-compose.yml

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,10 +13,7 @@ spec:
   replicas: 3
   selector:
     matchLabels:
-      app: sample-app
-  template:BASE: my_database
-      MYSQL_USER: my_user
```
Comment (devopseasylearning) [inline docker-compose.yml:18]: why this ?

**Comment by THIMONE-NOUPA** [comment]
Good! i think we can merge.

File: docker-compose.yml (hunk 1 above)
Comment (s7maryam) [reply docker-compose.yml:18]: i am also wondering

File: docker-compose.yml [hunk 2]
```diff
@@ -13,10 +13,7 @@ spec:
   replicas: 3
   selector:
     matchLabels:
-      app: sample-app
```
Comment (9A4119) [inline docker-compose.yml:16]: what were you trying to achieve here?

**Comment by EDEMOS80** [comment]
why this?

File: docker-compose.yml (hunk 1 above)
Comment (sdjakou) [reply docker-compose.yml:18]: hope to understand

Comment (Ajmensah) [inline docker-compose.yml:18]: @devopseasylearning  is that correct

File: docker-compose.yml [hunk 3]
```diff
@@ -13,10 +13,7 @@ spec:
...
-  template:BASE: my_database
```
Comment (s7rosine) [inline docker-compose.yml:17]: @S7belliss is this a good base image here

File: docker-compose.yml [hunk 4]
```diff
@@ -13,10 +13,7 @@ spec:
...
-      MYSQL_USER: my_user
-      MYSQL_PASSWORD: my_password
+  
     volumes:
       - db_data:/var/lib/mysql
```
Comment (sadioci2) [inline docker-compose.yml:18]: what are you trying to do here?

File: docker-compose.yml (hunk 1 above)
Comment (Perrine53) [reply docker-compose.yml:18]: we are moving to the right direction!!!

**Comment by THIMONE-NOUPA** [comment]
Good! i think we can merge.

File: docker-compose.yml [hunk 5]
```diff
@@ -13,10 +13,7 @@ spec:
   replicas: 3
   selector:
```
Comment (NtBahati) [inline docker-compose.yml:14]: Good job

File: docker-compose.yml (hunk 1 above)
Comment (nalukunyang) [reply docker-compose.yml:18]: good job

**Comment by s7maryam** [comment]
> why this?

okay

File: docker-compose.yml (hunk 1 above)
Comment (Djurizt) [reply docker-compose.yml:18]: Yes, good

File: docker-compose.yml [hunk 6]
```diff
@@ -13,10 +13,7 @@ spec:
...
     matchLabels:
```
Comment (omonefe08) [inline docker-compose.yml:15]: how to resolve

**Comment by rochdelaporte** [comment]
Great

**Pull request merged**